pkg/controller/constrainttemplate/constrainttemplate_controller_suite_test.go

```diff
@@ -50,6 +51,9 @@ func TestMain(m *testing.M) {
 	if err := apis.AddToScheme(scheme.Scheme); err != nil {
 		stdlog.Fatal(err)
 	}
+	if err := apiextensionsv1.AddToScheme(scheme.Scheme); err != nil {
```

Comment:
is this redundant given we've added apis/addtoscheme_customresourcedefinition_v1.go ?

Reply:
Yes, it is.  I forgot to remove after adding `apis/addtoscheme_customresourcedefinition_v1.go`. Good catch